jumpcloud-ADMU/Powershell/Tests/Public/invokeMigration.Tests.ps1

```diff
@@ -736,11 +742,8 @@ Describe "ADMU Bulk Migration Script CI Tests" -Tag "Migration Parameters" {
             # --- Modify Script Content in Memory ---
             # Change forceReboot to false
             $scriptContent = $scriptContent -replace '(\$ForceReboot\s*=\s*)\$true', '$1$false'
-
             # Change autoBindJCUser to false
             $scriptContent = $scriptContent -replace '(\$AutoBindJCUser\s*=\s*)\$true', '$1$false'
-            # Change forceReboot to false
```

Comment:
Why is this removed?

Reply:
Duplicate code:
<img width="447" height="509" alt="image" src="https://github.com/user-attachments/assets/21a6cc7e-6f28-4c1d-8000-04289f4e7959" />